student.rb

```diff
@@ -1,6 +1,6 @@
-# require_relative './person'
+require_relative './person'
 class Student < Person
-  def initialize(classroom)
+  def initialize(classroom, name, age)
     super(name, age)
     @classroom = classroom
   end
```

Comment:
- [ ] In the Student class, there should be a method that serves as the 'belongs-to' side (a student belongs to a classroom). Check out this article with examples: [both-ways-relationships](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#both-ways-relationships).